From 75b1a6971860b31283a4f9251bac88a4413e558b Mon Sep 17 00:00:00 2001
From: Patrick Yavitz <pyavitz@xxxxx.com>
Date: Sat, 18 Nov 2023 11:42:26 -0500
Subject: [PATCH] sun50i-h5-nanopi-neo-plus2: remove support for mem dynamic
 freq scaling

The introduction of DRAM devfreq support for Allwinner A64/H5
has made the unit unstable. So remove support for the driver.

https://lore.kernel.org/linux-arm-kernel/a48923b7-12f9-808e-1171-49b826bd5f1c@samsung.com/T/#ma00a3e07248dc7fb2d300b7c9c409f69ffa64c14

Signed-off-by: Patrick Yavitz <pyavitz@xxxxx.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-h5-nanopi-neo-plus2.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h5-nanopi-neo-plus2.dts b/arch/arm64/boot/dts/allwinner/sun50i-h5-nanopi-neo-plus2.dts
index 5e07a5352e76..1bda41aff547 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h5-nanopi-neo-plus2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h5-nanopi-neo-plus2.dts
@@ -89,6 +89,10 @@ rfkill_bt {
 	};
 };
 
+&mbus {
+	/delete-property/ compatible;
+};
+
 &cpu0 {
 	cpu-supply = <&vdd_cpux>;
 };
-- 
2.39.2

